embed.fnc

```diff
@@ -3373,6 +3373,16 @@ AMTdip	|void	|SvREFCNT_inc_void					\
 				|NULLOK SV *sv
 ARdp	|const char *|sv_reftype|NN const SV * const sv 		\
 				|const int ob
+
+Adp	|void	|sv_regex_global_pos_clear				\
+				|NN SV *sv
+Adp	|STRLEN |sv_regex_global_pos_get				\
```

Comment:
Less actionable,  I'd also consider changing the way it's called, currently you do:
```
STRLEN pos = sv_regex_global_pos_get(sv, 0);
if (pos != (STRLEN)-1)
    ...;
```
perhaps it could return whether the value is valid instead:
```
STRLEN pos;
if (sv_regex_global_pos_get(sv, &pos, 0)) 
   ... stuff with pos ...'
```
which I think has a better flow and avoids the magic number.

Reply:
Makes me wish for std::optional/expected (but no, I don't think we could switch to C++)